tools/cloudera/build_hue_common.sh

```diff
@@ -166,8 +177,8 @@ function sles12_install() {
       xmlsec1 xmlsec1-devel  xmlsec1-openssl-devel'
     # MySQLdb install
     sudo -- sh -c 'zypper install -y libmysqlclient-devel libmysqlclient18 libmysqld18 libmysqld-devel'
-    # NODEJS 14 install
-    sudo -- sh -c 'zypper install -y npm14 nodejs14'
+    # NODEJS 16 install
```

Comment:
Update comment to say why we use v 16

Reply:
Added a comment to explain why we are moving to v16.